-----------------------------------------------------------------
Revision: 225437b7c5b01a666461a56213713d77462490f9
Ancestor: f15338f57c0920b685e5164d2f53e50a6d0f64b7
Author: nosnilmot@pidgin.im
Date: 2008-07-01T20:47:28
Branch: im.pidgin.pidgin

Modified files:
        libpurple/protocols/oscar/oscar.h

ChangeLog: 

Fix ICQ login by mimicking the clientinfo versions used by the official
ICQ6 client.

============================================================
--- libpurple/protocols/oscar/oscar.h	a40e3332eade975c73e344e17bd06d4b99cae751
+++ libpurple/protocols/oscar/oscar.h	ab1fe743aec4287710fdb56eb412ee30f43ff318
@@ -258,6 +258,15 @@ struct _ClientInfo
 	"us", "en", \
 }
 
+#define CLIENTINFO_ICQ6_6_0_6059 { \
+	"ICQ Client", \
+	0x010a, \
+	0x0006, 0x0000, \
+	0x0000, 0x17ab, \
+	0x00007535, \
+	"us", "en", \
+}
+
 #define CLIENTINFO_ICQBASIC_14_3_1068 { \
 	"ICQBasic", \
 	0x010a, \
@@ -302,9 +311,9 @@ struct _ClientInfo
 #define CLIENTINFO_PURPLE_ICQ { \
 	"Purple/" VERSION, \
 	0x010a, \
-	0x0014, 0x0034, \
-	0x0000, 0x0bb8, \
-	0x0000043d, \
+	0x0006, 0x0000, \
+	0x0000, 0x17ab, \
+	0x00007535, \
 	"us", "en", \
 }
 
